ripemd/src/c320.rs

```diff
@@ -142,7 +142,7 @@ macro_rules! process_block(
     });
 );
 
-pub fn compress(h: &mut [u32; DIGEST_BUF_LEN], data: &[u8; 64]) {
+pub(crate) fn compress(h: &mut [u32; DIGEST_BUF_LEN], data: &[u8; 64]) {
```

Comment:
It's better to use either `pub(crate)` or `pub(super)` for both the function and the constants.

Reply:
Agreed, I've changed the remaining uses of `super` to basically just cover the architecture-specific compression functions as these are the only items which should definitely never be imported elsewhere in the crate.